DashKit/DashPumpManager.swift

```diff
@@ -1605,10 +1605,12 @@ extension DashPumpManager: PodCommManagerDelegate {
             let content = Alert.Content(title: LocalizedString("Pod System Error", comment: "Alert title for Pod System Error"),
                                         body: error.localizedDescription,
                                         acknowledgeActionButtonLabel: LocalizedString("OK", comment: "Alert acknowledgment OK button"))
+            let parameters = (try? JSONEncoder().encode(error)).flatMap { String(data: $0, encoding: .utf8) }
```

Comment:
Sorry, but I'm still not a fan of this.  I would like @ps2 to weigh in on serializing to JSON here.